lib/manager/terragrunt-version/readme.md

```diff
@@ -0,0 +1 @@
+This will maintain `.terragrunt-version` files. Available versions will be determined from gruntwork-io/terragrunt GitHub releases.
```

Comment:
```suggestion
This will maintain `.terragrunt-version` files.
Available versions will be determined from `gruntwork-io/terragrunt` GitHub releases.
```

- Use one sentence per line
- I think it looks nicer if we use the monospaced font for the reference to the `gruntwork-io/terragrunt` repository